### DISCUSSION TITLE
Questions on Nuxt 3 scaffold?

### DISCUSSION THREAD

**Discussion by fayazara** [nuxt/nuxt#18351] [Questions]
I tried Nuxt 3 beta today.

I used `npx nuxi init nuxt3-app`, it automatically created a package.json with some boilerplate files. 

1. Why does it create config files in typescript environment?
2. It uses Vite, but it mentions Vite is experimental in v3 [here](https://v3.nuxtjs.org/getting-started/introduction#comparison).

Nuxt 2 scaffold used to ask us if we want to use ts/js, UI framework to use, some useful nuxt modulesm like PWA, Axios etc. Will they be included in future releases?

**Comment by lucpotage** [comment]
Nuxt 3 is still in beta. I think there will be improvements around Vite support along the way. Regarding Typescript by default, I support the CLI will invite you to choose between TS and JS at some point too.
